src/styles/LoginStyles.ts

```diff
@@ -32,12 +34,15 @@ export const LoginButton = styled(Button)`
     box-shadow: none;
     width: 187px;
     height: 48px;
-    margin: 8px 0px;
     text-transform: none;
+    display: flex;
+    justify-content: center;
```

Comment:
This line doesn't seem to be doing anything (and also will be irrelevant if you change it to `display: block;`